docker/docker-entrypoint.sh

```diff
@@ -67,6 +67,10 @@ END
   echo "💡 Superuser Username: ${SUPERUSER_NAME}, E-Mail: ${SUPERUSER_EMAIL}"
 fi
 
+if [ "$NAUTOBOT_UWSGI_PROCESSES" == "true" ]; then
```

Comment:
This works because any non-zero value for this variable evaluates to true, do I have that right?